seeders/20190904224455-location_seeds.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('Locations', [
       {
-        id: 1,
+        id: 1000000,
```

Comment:
In refactor, we have got to find a severely different way of doing this. I'm glad we've figured out how to do this, how to make it work. But using seeds instead of mocks/stubs is problematic. Here's to finding a JS version of FactoryBot (maybe it works here too???)